packages/scheduler/lib/models/tasks.ts

```diff
@@ -184,19 +184,37 @@ export async function transitionState(
     }
 
     const output = 'output' in props ? props.output : null;
+    const asPostgresJson = (val: JsonValue) => {
+        if (val === null) {
+            return null;
+        }
+        if (Array.isArray(val)) {
+            // https://github.com/brianc/node-postgres/issues/442
```

Comment:
ahah I have this exact comment in a 8 years old codebase 😬

Reply:
😬 and 😢